core/src/main/java/io/questdb/network/Net.java

```diff
@@ -230,6 +230,10 @@ public static int getTcpNoDelay(long fd) {
         return getTcpNoDelay(toOsFd(fd));
     }
 
+    public static void init() {
```

Comment:
Debris?

Reply:
An empty method to force-load the class. -> the class can initialize (and allocate) its static state before we start tests. If we initialize the class only after starting the test then `assertMemLeak()` will report a leak. 

I think I saw this pattern elsewhere in the code.

Reply:
that is correct, it might be prudent to call Net.init() from Os.init() because they use the same binary library and Os.init() is already being called from a lot of places